src/ising.jl

```diff
@@ -1,8 +1,45 @@
 export ising_graph, energy
 export gibbs_tensor, brute_force
-export State
+export State, Cluster, Spectrum
 
 const State = Union{Vector, NTuple}
+const Instance = Union{String, Dict}
+const EdgeIter = Union{LightGraphs.SimpleGraphs.SimpleEdgeIter, Base.Iterators.Filter}
+
+struct Spectrum
```

Comment:
`Spectrum` again? Is this correct that:
1. `PEPS.jl` is not inclunded
2. `Spectrum` defined here is used